src/envoy/mixer/repositories.bzl

```diff
@@ -18,10 +18,9 @@
 MIXER_CLIENT = "847a67bd905605f87b3e9a559fca905434fb50c2"
 
 def mixer_client_repositories(bind=True):
```

Comment:
?

Reply:
This will be changed after Mixerclient change is submitted.